### PR TITLE
All styling changes

### PR DESCRIPTION
**Pull request by kevinshare**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,2 @@
 node_modules
 .env
-router.js
-server.test.js
```
Comment (ZeMunchkin) [inline .gitignore]: new line

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon ./server.js"
+    "start": "nodemon ./server.js",
```
Comment (ZeMunchkin) [inline package.json]: you'll want to make this start script be node and not nodemon

File: public/style.css [hunk 3]
```diff
@@ -1,30 +1,202 @@
...
+    display: block;
+    text-align: left;
+    width: 100%;
+    margin: 0;
+    padding: 14px;
+  }
+  .topnav input[type=text] {
+      border: 1px solid #ccc;
+  }
 }
```
Comment (ZeMunchkin) [inline public/style.css]: new line

File: router.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  request(url).pipe(res);
+});
+router.post('/', function(req,res) {
+  const path = req.params.restaurantId;
+  const url = `http://localhost:3001/restaurants/${path}/reviews`;
+  request(url).pipe(res);
+});
+
+
+module.exports = router;
```
Comment (ZeMunchkin) [inline router.js]: new line. Make sure you're using a linter

**Review by ZeMunchkin — CHANGES_REQUESTED** [review]
Double check that you have all your linter errors resolved

File: .gitignore [hunk 5]
```diff
@@ -1,2 +1,5 @@
 node_modules
 .env
+router.js
+server.js
+server.test.js
```
Comment (MichaelDurfey) [inline .gitignore:5]: You should just gitignore the file if you want to leave out a test..?

**Review by MichaelDurfey — APPROVED** [review]

**Pull request merged**